cuckoo/cuckoo_result.py

```diff
@@ -1662,7 +1662,6 @@ def _create_process_tree_hashes(process_tree: List[Dict[str, Any]]) -> List[str]
 
     return process_tree_hashes
 
-
```

Comment:
Since these methods are not in a class but rather in a module, they require 2 blank lines between them